plugins/inputs/docker_log/README.md

```diff
@@ -0,0 +1,59 @@
+# Docker Log Input Plugin
+
+The docker log plugin uses the Docker Engine API to get logs on running
+docker containers.
+
+The docker plugin uses the [Official Docker Client](https://github.com/moby/moby/tree/master/client)
+to gather logs from the [Engine API](https://docs.docker.com/engine/api/v1.24/).
+Note: This plugin works only for containers with the `json-file` or `journald` logging driver.
```

Comment:
Maybe this also works with `local`?  I also wonder if Docker Enterprise would support this using "dual logging"?  @fntlnz Do you know or have a guess?

https://docs.docker.com/config/containers/logging/configure/#limitations-of-logging-drivers

Reply:
Add local to readme . Not sure about the dual logging aspects of it.